<launch>
    <include file="$(find vrpn_client_ros)/launch/all_mocap_vrpn.launch" />
    <node pkg="tf" type="static_transform_publisher" name="map2word" args="0 0 0 0 0 0 map world 100" />
    <node pkg="mocap_pose" name="mocap_pose_node" type="mocap_pose_node" output="screen">
        <param name="vrpn_topic" value="/vrpn_client_node/BUAA_06/pose"/>
        <param name="mavros_topic" value="/mavros/vision_pose/pose"/>
        <!--param name="mavros_topic" value=""/-->
        <param name="drone_pose_topic" value="/drone_mocap/pose" />
        <param name="drone_odom_topic" value="/drone_mocap/odom" />
        <param name="offset_x" value = "0.0" />
        <param name="offset_y" value = "0.0" />        
    </node>
</launch>
